src/activities/Elsa.Activities.AzureServiceBus/Activities/SendAzureServiceBusMessage/SendAzureServiceBusTopicMessage.cs

```diff
@@ -0,0 +1,42 @@
+using System.Text;
+using System.Threading.Tasks;
+using Elsa.Activities.AzureServiceBus.Services;
+using Elsa.ActivityResults;
+using Elsa.Attributes;
+using Elsa.Serialization;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Microsoft.Azure.ServiceBus;
+
+namespace Elsa.Activities.AzureServiceBus
+{
+    [Action(Category = "Azure Service Bus", DisplayName = "Send Service Bus Topic Message", Description = "Sends a message to the specified topic", Outcomes = new[] { OutcomeNames.Done })]
+    public class SendAzureServiceBusTopicMessage : Activity
+    {
+        private readonly ITopicMessageSenderFactory _messageSenderFactory;
+        private readonly IContentSerializer _serializer;
+
+        public SendAzureServiceBusTopicMessage(ITopicMessageSenderFactory messageSenderFactory, IContentSerializer serializer)
+        {
+            _messageSenderFactory = messageSenderFactory;
+            _serializer = serializer;
+        }
+
+        [ActivityProperty] public string TopicName { get; set; } = default!;
+        [ActivityProperty] public object Message { get; set; } = default!;
+
+        protected override async ValueTask<IActivityExecutionResult> OnExecuteAsync(ActivityExecutionContext context)
+        {
+            var sender = await _messageSenderFactory.GetTopicSenderAsync(TopicName, context.CancellationToken);
+            var json = _serializer.Serialize(Message);
```

Comment:
As per your suggestion, perhaps here we might first perform a check if the `Message` is a string value. If anything else, it gets serialized, otherwise sent as-is.

Reply:
yep , I'll change this part to be sure that "simple" work (need to do on Sender And Receiver.